src/Compilers/CSharp/Test/Symbol/Symbols/AssemblyAndNamespaceTests.cs

```diff
@@ -471,5 +625,166 @@ namespace N { }
                 // (2,1): error CS1671: A namespace declaration cannot have modifiers or attributes
                 Diagnostic(ErrorCode.ERR_BadModifiersOnNamespace, "[System.Obsolete]").WithLocation(1, 1));
         }
+
+        private static readonly CSharpParseOptions s_previewOptions = CSharpParseOptions.Default.WithLanguageVersion(LanguageVersion.Preview);
```

Comment:
nit: Typically we use `TestOptions.RegularPreview`

Reply:
Renamed this. I think it could be handy to be able to change this from Preview to Regular in one place. We have done this for other features in the past, IIRC.